src/chrome/content/rules/ComputerWoche.de.xml

```diff
@@ -0,0 +1,47 @@
+<!-- 
+	Cert mismatch:
+		- bestincloud-de.computerwoche.de
+		- blog.computerwoche.de
+		- download.computerwoche.de
+		- gala.computerwoche.de
+		- beta.ibmexperts.computerwoche.de
+		- idgbusmedia.computerwoche.de
+		- iphonetest.computerwoche.de
+		- jn.computerwoche.de
+		- juk.computerwoche.de
+		- karrierewiki.computerwoche.de
+		- mobil.computerwoche.de
+		- preisvergleich.computerwoche.de
+		- testevent.computerwoche.de
+		- video.computerwoche.de
+		- w3.computerwoche.de
+		- wiki.computerwoche.de
+		- ww.computerwoche.de
+		- www1.computerwoche.de
+		- www2.computerwoche.de
+		- www3.computerwoche.de
+		- wwwl.computerwoche.de
```

Comment:
@numismatika could you add a hint about the wildcard DNS record and maybe you can also remove some of the `www`aliases from the list.